opel-collector/values.yaml

```diff
@@ -0,0 +1,95 @@
+opentelemetry-collector:
+  mode: daemonset
+
+  fullnameOverride: otel-agent
+
+  extraEnvs:
+  - name: CORALOGIX_PRIVATE_KEY
+    valueFrom:
+      secretKeyRef:
+        name: coralogix-otel-privatekey
+        key: PRIVATE_KEY
+  - name: APP_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+  - name: SUBSYSTEM
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+  - name: KUBE_NODE_NAME
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: spec.nodeName
+  - name: CORALOGIX_ENDPOINT
+    value: "tracing-ingress.eu2.coralogix.com"
+    
+  config:
+    exporters:
+      coralogix:
+        endpoint: "${CORALOGIX_ENDPOINT}:9443"
+        private_key: "${CORALOGIX_PRIVATE_KEY}"
+        application_name: "${APP_NAME}"
+        subsystem_name: "${SUBSYSTEM}"
+    receivers:
+      prometheus: null
+    processors:
```

Comment:
```
      batch: {}
      k8sattributes:
        passthrough: false
        extract:
          labels:
          - key_regex: '(.*)'
      memory_limiter: null
```

Reply:
why we dont want the filter and all of the metadata I used ?

Reply:
oh ok I understand, its taking all of the labels instead only specific ones. 
And I still kept the `node_from_env_var` filter because its supposed to reduce the resources usage [by querying only the pods running with the agent on the same host] so I think we should use it